library/core/src/marker.rs

```diff
@@ -429,6 +429,13 @@ marker_impls! {
 
 }
 
+// todo: which feature gate should we use?
+#[cfg(not(bootstrap))]
+marker_impls! {
+    #[stable(feature = "rust1", since = "1.0.0")]
+    Copy for f16, f128
+}
+
```

Comment:
Which feature gates should be used for trait impls? Most of the macros (not this specific example) don't allow setting a feature gate so all traits just get marked `rust1` by default. I could adjust the other macros to take a gate attribute like this one, but it seems pointless.